src/main.js

```diff
@@ -33,9 +25,18 @@ let load = base.loader({
     }
   },
 
+  monitor: {
+    requires: ['process', 'profile', 'cfg'],
+    setup: ({process, profile, cfg}) => base.monitor({
+      project: cfg.monitor.component,
```

Comment:
I would suggest project: taskcluster-treeherder-staging for the staging environment... I think that's how we so with everything else... nothing should be shared between production and staging deployment...